Makefile

```diff
@@ -1,11 +1,24 @@
 ROOT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 
+# All components are prefixed by st2
+COMPONENTS := $(wildcard /tmp/st2/st2*)
```

Comment:
Correct, without adding st2 components to the pack, pylint is not aware of the base `Action` and `Sensor` class signatures, etc.